extensions/guacamole-auth-ldap/src/main/java/org/apache/guacamole/auth/ldap/group/UserGroupService.java

```diff
@@ -79,17 +79,17 @@
      * @throws GuacamoleException
      *     If guacamole.properties cannot be parsed.
      */
-    private ExprNode getGroupSearchFilter() throws GuacamoleException {
+    private ExprNode buildGroupSearchFilter() throws GuacamoleException {
```

Comment:
Why is this being renamed?

Reply:
I though it was more descriptive this name rather than get, also to avoid confusion with 
`        return confService.getGroupSearchFilter();
`